omod/src/main/java/org/openmrs/module/fhir/providers/RestfulObservationResourceProvider.java

```diff
@@ -75,6 +75,17 @@ public Observation getResourceById(@IdParam IdType theId) {
 		return provider.searchObsById(id);
 	}
 
+	/**
+	 * @see org.openmrs.module.fhir.resources.FHIRObservationResource#searchObsByPatientAndCode(ca.uhn.fhir.rest.param.ReferenceParam, ca.uhn.fhir.rest.param.TokenOrListParam)
+	 */
+	@Search
+	public List<Observation> findObsByPatientAndCode(@RequiredParam(name = Observation.SP_SUBJECT) ReferenceParam
+			person,
+			@RequiredParam(name = Observation.SP_CODE) TokenOrListParam
+					codes) {
+		return provider.searchObsByPatientAndCode(person, codes);
```

Comment:
Where is `provider` coming from?

Reply:
It was already defined
`private FHIRObservationResource provider;`